host_source.go

```diff
@@ -539,12 +537,16 @@ func (t *TabletInfo) Replicas() []ReplicaInfo {
 	return t.replicas
 }
 
+type TabletInfoList struct {
```

Comment:
Why not have it as a plain custom type ?
```suggestion
type TabletInfoList []*TabletInfo
```